_posts/2015-09-08-code-talks.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "ePages talks REST API"
+date: "2015-09-08 08:04:00"
+icon: code
+tags: misc
+categories: tech-stories conferences api
+authors: ["Birgit"]
+---
+
+Have you signed up for `code.talks` ?
+Would you like to learn more about the challenges of documenting REST APIs?
+We will be happy to see you on our talk then.
```

Comment:
I would add some more details of the talk. Who is speaking? When and where is it taking place? Maybe also give more info why you chose this topic and then move on to the key points.